source/solvers/navier_stokes_vof_assemblers.cc

```diff
@@ -31,16 +31,16 @@ GLSNavierStokesVOFAssemblerCore<dim>::assemble_matrix(
 
   std::vector<double> &phase_values = scratch_data.phase_values;
 
-  // Limit force application : not applied if cell density is below
-  // density_ratio of the maximum density (e.g. when one of the fluids is air)
-  const double density_ratio = 2;
-  double       phase_cutoff  = 0;
-
   Assert(
     scratch_data.properties_manager.density_is_constant(),
     RequiresConstantDensity(
       "GLSVansAssemblerDiFelice<dim>::calculate_particle_fluid_interactions"));
 
+  // Limit force application : not applied if cell density is below
+  // density_ratio of the maximum density (e.g. when one of the fluids is air)
+  const double density_ratio = 2;
+  double       phase_cutoff  = 0;
```

Comment:
I don't like this. I think it's just better to remove this force limitation mechanism right now. This is not a good idea. Let's just apply the force everywhere and be done with it... This is going to be removed ASAP otherwise and it's not a good idea. Force is force, it applies to both fluid, let's be done with it.

Reply:
Limiting the source term is removed from the VOF assemblers.

Reply:
so the `density_ratio` and `phase_cutoff` are removed as well? It does not appear as such on git

Reply:
Yeah they should be removed no?